.github/workflows/update-schema-dependency-version.yaml

```diff
@@ -1,37 +1,52 @@
-name: Update Version
+name: Update Schema Version
 
 on:
   workflow_call:
     inputs:
       major:
         description: 'Schema Major version'
+        type: string
         required: true
       minor:
         description: 'Schema Minor version'
+        type: string
         required: true
       patch:
         description: 'Schema Patch version'
+        type: string
+        required: true
+      downstream-repo-to-update:
+        description: 'The name of the downstream repository for which to update the schema version'
+        type: string
+        required: true
+      schema-property-name-in-repo:
+        description: 'The property name in the repos pom.xml for the schema version, e.g. rcsb.mojave.model.version'
+        type: string
         required: true
 
 
 env:
-  NEW_SCHEMA_VERSION: "${{ github.event.inputs.major }}.${{ github.event.inputs.minor }}.${{ github.event.inputs.patch }}"
+  NEW_SCHEMA_VERSION: "${{ inputs.major }}.${{ inputs.minor }}.${{ inputs.patch }}"
 
 jobs:
   update-version:
     runs-on:
       - self-hosted
       - java
     steps:
-      - uses: actions/checkout@v3
+      - name: Checking out ${{ inputs.downstream-repo-to-update }}
+        uses: actions/checkout@v3
+        with:
+          repository: rcsb/${{ inputs.downstream-repo-to-update }}
+          path: ${{ inputs.downstream-repo-to-update }}
       - name: Setup git author information
         run: |
```

Comment:
It might be you are missing `cd` here, no?

Reply:
Don't know, I thought `path` would be for that? The docs are not that clear.

Reply:
OK, thanks, just try then :)